client/consensus/babe/src/aux_schema.rs

```diff
@@ -25,10 +25,11 @@ use sc_client_api::backend::AuxStore;
 use sp_blockchain::{Result as ClientResult, Error as ClientError};
 use sp_runtime::traits::Block as BlockT;
 use sp_consensus_babe::BabeBlockWeight;
-use sc_consensus_epochs::{EpochChangesFor, SharedEpochChanges};
+use sc_consensus_epochs::{EpochChangesFor, SharedEpochChanges, migration::EpochChangesForV0};
 use crate::Epoch;
 
-const BABE_EPOCH_CHANGES: &[u8] = b"babe_epoch_changes";
+const BABE_EPOCH_CHANGES_V0: &[u8] = b"babe_epoch_changes";
```

Comment:
In GRANDPA code we are keeping the data under the same key, but instead we keep track of data version in a different key (https://github.com/paritytech/substrate/blob/master/client/finality-grandpa/src/aux_schema.rs#L37). I think this is a bit tidier but wouldn't block on this.